src/main/java/org/springframework/cloud/deployer/spi/cloudfoundry/AbstractCloudFoundryDeployer.java

```diff
@@ -175,6 +176,21 @@ Path getApplication(AppDeploymentRequest request) {
 			.doOnError(e -> logger.error(String.format("Retry operation on getStatus failed for %s.  Max retry time %sms", id, statusTimeout)));
 	}
 
+	protected void deleteLocalApplicationResourceFile(AppDeploymentRequest appDeploymentRequest) {
+		try {
+			if (!appDeploymentRequest.getResource().getURI().getScheme().toLowerCase().startsWith("http")) {
```

Comment:
condition is backwards?

Reply:
sorry, this was a mistake when I changed the code. Thanks for fixing it when merging.